index.html

```diff
@@ -32,6 +32,13 @@
       specStatus: "CG-DRAFT",
       group: "wicg",
       localBiblio: {
+        "ISO18013-5": {
```

Comment:
FWIW: https://github.com/tobie/specref/issues/776 ... we might want to just send this to specref, as I've not managed to get an answer as to why it's not showing up in SpecRef... 

Also, we purposely avoided mentioning mDoc until we get things into the registry 🙈

Reply:
Agreed that it belongs in SpecRef, I searched for it there first and didn't find it, so added it manually. 

We can use the SpecRef reference when it shows up there.

Reply:
Yeah, I'll send a PR if the above doesn't get fixed... actually, I'll just send it now...

Reply:
https://github.com/tobie/specref/pull/779

Reply:
Ok, I'll drop the local ref presuming tobie/specref#779 will go in soon.